activities/Blockrain.activity/lib/tutorial.js

```diff
@@ -0,0 +1,91 @@
+define([], function () {
+	var tutorial = {};
+
+	tutorial.start = function() {
+		var steps = [
+			{
+				element: "",
+				orphan: true,
+				placement: "bottom",
+				title: "Blockrain",
```

Comment:
All title and content should be localized as explained [here](https://github.com/llaske/sugarizer/blob/dev/docs/tutorial_step9.md#localize-the-tutorial).